source/plugin/basics/entities/spawning.rst

```diff
@@ -11,22 +11,24 @@ For example, let's try to spawn a Creeper:
 
     import org.spongepowered.api.entity.EntityTypes;
     import org.spongepowered.api.world.Location;
-    import org.spongepowered.api.world.World;
+    import org.spongepowered.api.world.extent.Extent;
 
     import com.google.common.base.Optional;
 
     public void spawnEntity(Location location) {
-        World world = location.getWorld();
+        Extent extent = location.getExtent();
          // We need to create the entity
-        Optional<Entity> optional = world.createEntity(EntityTypes.CREEPER, location);
+        Optional<Entity> optional = extent.createEntity(EntityTypes.CREEPER, location.getPosition());
         if (optional.isPresent()) {
             // After this, we can use more API that relates to creeper
             Creeper creeper = (Creeper) optional.get();
-            creeper.setPowered(true);
-            creeper.setExplosionRadius(10);
+            ExplosiveRadiusData radiusData = creeper.getExplosiveRadiusData();
+            radiusData.setExplosionRadius(10);
+            // Return the modified explosive radius data back to the creeper
+            creeper.offer(radiusData);
```

Comment:
Why not add `ChargedData` and `DisplayNameData` to show a bit more of an example?